user_scripts/wrangler_by_sample.yaml

```diff
@@ -24,9 +24,7 @@ cpu_count: 5
 memory_mb_per_step: 20000
 
 #location of sample sheet
-input_sample_sheet_directory: /home/charlie/projects/miptools_data/miptools_test-data/test_data
-
-input_sample_sheet_name: sample_list.tsv
+input_sample_sheet: /home/charlie/projects/miptools_data/miptools_test-data/test_data/sample_list.tsv
```

Comment:
This is a nice fix - decreases the number of variables a user has to enter into the yaml file